src/model/Computation.js

```diff
@@ -62,6 +62,9 @@ export default class Computation extends Model {
 
 		this.boundsSensitive = true;
 		this.dirty = true;
+
+		// computations don't shuffle
+		this.shuffle = undefined;
```

Comment:
I was kinda thinking a simpler `Model` base class that could be used instead of `Model` for some of the more esoteric `Model` subclasses.